app/builders/immediate_email_builder.rb

```diff
@@ -1,43 +1,57 @@
 class ImmediateEmailBuilder
-  def initialize(subscriber:, content_change:)
-    @subscriber = subscriber
-    @content_change = content_change
+  def initialize(subscriber_content_changes)
+    @subscriber_content_changes = subscriber_content_changes
   end
 
   def self.call(*args)
     new(*args).call
   end
 
   def call
-    {
-      address: subscriber.address,
-      subject: subject,
-      body: body
-    }
+    Email.import!(records)
```

Comment:
I notice this creates the emails, whereas I think the digest one merely builds one. Might it make sense for the digest one to also create for consistency?